pkg/relayer/proxy/proxy.go

```diff
@@ -108,20 +107,19 @@ func NewRelayerProxy(
 		deps,
 		&rp.clientCtx,
 		&rp.blockClient,
+		&rp.accountsQuerier,
+		&rp.supplierQuerier,
+		&rp.sessionQuerier,
+		&rp.applicationQuerier,
```

Comment:
See the comment above (https://github.com/pokt-network/poktroll/pull/199/files#r1405127660) this should probably be interfaces that abstract the chain interaction.

See https://github.com/pokt-network/poktroll/pull/187/files#diff-93eab163375000803044e29070dbef1b1203622442eae9619001e19019ee1afe:~:text=type-,AccountQueryClient,-interface%20%7B for examples of the ones from #187